widget/playground/src/constants.ts

```diff
@@ -38,3 +40,123 @@ export enum SIDE_TABS_IDS {
 export const DEFAULT_PRIMARY_RADIUS = 20;
 export const DEFAULT_SECONDARY_RADIUS = 25;
 export const NOT_FOUND = -1;
+export const THEME: { id: Mode; title: string }[] = [
+  {
+    id: 'light',
+    title: 'Light',
+  },
+
+  {
+    id: 'dark',
+    title: 'Dark',
+  },
+
+  {
+    id: 'auto',
+    title: 'System',
+  },
+];
+
+export const CUSTOM_THEMES = [
```

Comment:
I suggest to move it to another file for playground presets. Also please declare its type to make it more readable.

Reply:
fixed